ibm/data_source_ibm_is_security_group_rule_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccIBMIsSecurityGroupRuleDataSourceBasic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccCheckIBMIsSecurityGroupRuleDataSourceConfigBasic(),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("data.ibm_is_security_group_rule.is_security_group_rule", "id"),
+					resource.TestCheckResourceAttrSet("data.ibm_is_security_group_rule.is_security_group_rule", "security_group_id"),
+					resource.TestCheckResourceAttrSet("data.ibm_is_security_group_rule.is_security_group_rule", "id"),
```

Comment:
no need to check IDs are set as they are marked Required

Reply:
removed.